modules/ossm-threescale-configuring-the-threescale-webassembly-module.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assembly:
+//
+// service_mesh/v2x/ossm-threescale-webassembly-module.adoc
+
+[id="threescale-configuring-the-threescale-webassembly-module_{context}"]
+= Configuring the 3scale WebAssembly module
+
+The architecture of the 3scale WebAssembly module configuration depends on the 3scale account and authorization service, and the list of services to handle.
+
+.Prerequisites
+
+The prerequisite are a set of minimum mandatory fields in all cases:
```

Comment:
s/prerequisite are/prerequisites are/

Reply:
Is it possible to form everything under the `.Prerequisite` heading here in unordered list format?